{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_views.py

```diff
@@ -1,8 +1,14 @@
 import pytest
 from django.conf import settings
 from django.test import RequestFactory
+from django.urls import reverse
 
-from {{ cookiecutter.project_slug }}.users.views import UserRedirectView, UserUpdateView
+from {{ cookiecutter.project_slug }}.users.tests.factories import UserFactory
+from {{ cookiecutter.project_slug }}.users.views import (
+    UserDetailView
```

Comment:
My bad, forgot a trailing comma in my previous suggestion:

```suggestion
    UserDetailView,
```

Reply:
I appended lower() to the path() strings and I think the test now demonstrates that there's a bug. Note that I have two versions of the test, one that calls the view directly and one that uses the test client fixture to traverse through, well, everything.

I added the comma ;).

Reply:
I meant to say that the first test passes and the second test fails. So that suggests that the bug isn't in the view.